src/test/java/edu/ucsb/ucsbcslas/entities/ActiveQuarterTests.java

```diff
@@ -0,0 +1,48 @@
+package edu.ucsb.ucsbcslas.entities;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import org.junit.jupiter.api.Test;
+
+public class ActiveQuarterTests {
+  @Test
+  public void testQuarter_toString() {
+    ActiveQuarter q = new ActiveQuarter(1L, "f20");
+    assertEquals("{ id='1', activeQuarter='f20'}", q.toString());
+
+    Admin admin2 = new Admin("admin@test.org");
+    assertEquals("Admin[ id=0, email=admin@test.org, isPermanentAdmin=false ]", admin2.toString());
+  }
+
+  @Test
+  public void testActiveQuarter_getAndSetId() {
+    ActiveQuarter q = new ActiveQuarter();
+    q.setId(4L);
+    assertEquals(4L, q.getId());
+  }
+
+  @Test
+  public void testActiveQuarter_notEqualNull() {
+    ActiveQuarter q = new ActiveQuarter();
+    assertNotEquals(q, null);
+  }
+
+  @Test
+  public void testActiveQuarter_notEqualDifferentClass() {
+    ActiveQuarter q = new ActiveQuarter();
+    assertNotEquals(q, new Object());
+  }
+
+  @Test
+  public void testAdmin_equalsSelf() {
+    ActiveQuarter q = new ActiveQuarter();
+    assertEquals(q, q);
+  }
+
+  @Test
+  public void testAdmin_equalsCopy() {
+    ActiveQuarter q1 = new ActiveQuarter();
+    ActiveQuarter q2 = new ActiveQuarter();
+    assertEquals(q1, q2);
+  }
```

Comment:
Check out the tests on the `main` branch for how to reduce these boilerplate getter and setter tests: https://github.com/ucsb-cs156-f20/proj-ucsb-cs-las/blob/eabfd2fb7643903c7429a4bceb27726255e7f445/src/test/java/edu/ucsb/ucsbcslas/entities/OnlineOfficeHoursTests.java#L14